python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -659,6 +663,45 @@ def update_effective_time_stats(self, summary, exposure):
         summary.effTimeSkyBgScale = float(b_eff)
         summary.effTimeZeroPointScale = float(c_eff)
 
+    def update_magnitude_limit_stats(self, summary, exposure, snr=5):
+        """Compute summary statistics related to the magnitude limit of a
+        point-like source.
+
+        The magnitude limit is calculated at a given SNR from the PSF,
+        sky, zeropoint, and readnoise in accordance with SMTN-002 [1]_
+        and LSE-40 [2]_.
+
+        References
+        ----------
+
+        .. [1] "Calculating LSST limiting magnitudes and SNR" (SMTN-002)
+        .. [2] "Photon Rates and SNR Calculations" (LSE-40)
+
+
+        Parameters
+        ----------
+        summary  : `lsst.afw.image.ExposureSummaryStats`
+            Summary object to update in-place.
+        exposure : `lsst.afw.image.ExposureF`
+            Exposure to grab band and exposure time metadata
+        snr      : `float`, optional
+        """
+        if exposure.getDetector() is None:
+            summary.magLim = float("nan")
+            return
+
+        # Calculate the readnoise
+        readNoiseList = list(ipIsr.getExposureReadNoises(exposure).values())
```

Comment:
There is an outstanding issue here that I don't know how to verify the read noise units in the old ISR task are actually the image units (`LSST ISR UNITS`)